setup.py

```diff
@@ -38,6 +38,9 @@
     if os.path.exists(pjoin(d, '__init__.py')):
         packages.append(d[len(here)+1:].replace(os.path.sep, '.'))
 
+package_data = {'nb_conda_kernels': [pjoin('logos', 'python', '*'),
```

Comment:
Even more so: some of the kernels ship a [kernel.js](https://github.com/gibiansky/IHaskell/blob/master/html/kernel.js)!